run-export.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+DOCS=markdown/en
+API=${DOCS}/api
+OF=$(pwd)/export
```

Comment:
Is absolute path required for export? If yes, can we use $PWD instead of evaluating pwd command?